packages/redux/src/orders/reducer.ts

```diff
@@ -9,26 +9,28 @@ import {
 import merge from 'lodash/merge';
 import omit from 'lodash/omit';
 import produce from 'immer';
-import reducerFactory from '../helpers/reducerFactory';
+import reducerFactory, {
+  createReducerWithResult,
+} from '../helpers/reducerFactory';
 import type * as T from './types';
 import type {
-  FetchOrderReturnOptionsSuccessAction,
-  FetchOrderReturnsSuccessAction,
+  // FetchOrderReturnOptionsSuccessAction,
+  // FetchOrderReturnsSuccessAction,
   ResetOrderDetailsStateAction,
   ResetOrderReturnOptionsStateAction,
-  ResetOrderReturnsEntitiesAction,
+  // ResetOrderReturnsEntitiesAction,
   ResetOrderReturnsStateAction,
 } from './types';
-import type {
-  OrderEntity,
-  ReturnEntity,
-  ReturnOptionEntity,
-} from '../entities/types';
+// import type {
+//   // OrderEntity,
+//   // ReturnEntity,
+//   ReturnOptionEntity,
+// } from '../entities/types';
 import type { StoreState } from '../types';
 
 export const INITIAL_STATE: T.OrdersState = {
-  error: null,
-  isLoading: false,
+  error: {},
+  isLoading: {},
   result: null,
```

Comment:
`result` can be an object as well since it will not contain a single result but all hashed results.

Reply:
Fixed 👍